internal/provider/resource_cloudflare_access_application_test.go

```diff
@@ -103,6 +105,36 @@ func TestAccCloudflareAccessApplication_WithCORS(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareAccessApplication_WithSass(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_application.%s", rnd)
+	accountID := os.Getenv("CLOUDFLARE_ACCOUNT_ID")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccessAccPreCheck(t)
+		},
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckCloudflareAccessApplicationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudflareAccessApplicationConfigWithSaas(rnd, accountID),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(name, "account_id", accountID),
+					resource.TestCheckResourceAttr(name, "name", rnd),
+					resource.TestCheckResourceAttr(name, "type", "saas"),
+					resource.TestCheckResourceAttr(name, "session_duration", "24h"),
+					resource.TestCheckResourceAttr(name, "saas_app.#", "1"),
+					resource.TestCheckResourceAttr(name, "saas_app.0.sp_entity_id", "1"),
+					resource.TestCheckResourceAttr(name, "saas_app.0.consumer_service_url", "1"),
+					resource.TestCheckResourceAttr(name, "saas_app.0.name_id_format", "1"),
```

Comment:
need to fix these assertions

```suggestion
					resource.TestCheckResourceAttr(name, "saas_app.0.sp_entity_id", "saas-app.example"),
					resource.TestCheckResourceAttr(name, "saas_app.0.consumer_service_url", "https://saas-app.example/sso/saml/consume"),
					resource.TestCheckResourceAttr(name, "saas_app.0.name_id_format", "email"),
```

Reply:
if you fix these, the acceptance tests all pass